src/app/components/site/cs/HomepageCS.tsx

```diff
@@ -1,233 +1,201 @@
+                            {isLoggedIn(user) && isTeacherOrAbove(user) &&
+                                <Button color={"dark-primary"} tag={Link} to={"/dashboard"}>Go to My Ada Overview</Button>}
+                        </div>
                     </div>
-                    }
                 </Container>
-            </section>
-            <section id="benefits-for-teachers-and-students" className="bg-white">
-                <Container className={"homepage-padding mw-1600"}>
-                    <Row className={"align-items-center"}>
-                        <Col xs={12} lg={5} className="mt-4 mt-lg-4 order-1 order-lg-0">
-                            <picture>
-                                <source srcSet="/assets/cs/decor/benefits-for-homepage.png" type="image/png"/>
-                                <img className={"d-block w-100 mw-760 px-sm-7 px-lg-0"} src={"/assets/cs/decor/benefits-for-homepage.png"} alt="" />
-                            </picture>
-                        </Col>
-                        <Col xs={12} lg={7}>
-                            <h2 className={"font-size-1-75 mb-4"}>What we offer</h2>
-                            <ul className={"font-size-1 font-size-md-1-25"}>
-                                <li>Free computer science resources: Tailored for students aged 14 to 19</li>
-                                <li>Interactive questions: Over 1000 questions with instant marking and feedback</li>
-                                <li>Teacher tools: Set quizzes and assignments effortlessly</li>
-                                <li>AI and machine learning resources: Stay ahead of the AI curve</li>
-                                <li>Complete curriculums: For
-                                    {" "}<a href={"/exam_specifications_england#gcse/aqa"}>GCSE</a>,
-                                    {" "}<a href={"/exam_specifications_england#a_level/aqa"}>A&nbsp;Level</a>,
-                                    {" "}<a href={"/exam_specifications_scotland#scotland_national_5/sqa"}>National&nbsp;5</a>,
-                                    {" "}<a href={"/exam_specifications_scotland#scotland_higher/sqa"}>Higher</a>, and
-                                    {" "}<a href={"/exam_specifications_scotland#scotland_advanced_higher/sqa"}>Advanced&nbsp;Higher</a></li>
-                            </ul>
-                            <div>
-                                <Button className="mt-3 me-3" tag={Link} to="/students" color="solid">Student Resources</Button>
-                                <Button className="mt-3" tag={Link} to="/teachers" color="solid">Teacher Resources</Button>
-                            </div>
-                        </Col>
-                    </Row>
+                <Container className={"mw-1600 homepage-padding-x"} fluid>
+                    <Card id={"cta-features-card"} className={"icon-card p-5"}>
+                        <Row className={"justify-content-center gy-5"}>
```

Comment:
These are bold in the designs:
```suggestion
                        <Row className={"justify-content-center gy-5 fw-bold"}>
```